.gitignore

```diff
@@ -30,6 +30,9 @@ DerivedData/
 timeline.xctimeline
 playground.xcworkspace
 
+## secrets
+ui-sdk-components-sample/UISDK*components*sample/Secrets.swift
```

Comment:
We need to add more information about how to setup the project and the format for the Secrets.swift